### ISSUE TITLE
IE11 - Crytpo is undefined

### ISSUE THREAD

**Issue by paulm999** [prebid/Prebid.js#3076]
## Type of issue
Bug

## Description
When using the consent module in prebid on ie11 or earlier, the auction fails to run as it throws a 'crypto' is undefined error. 

## Platform details
Prebid.js 1.22 , 1.23

**Comment by paulm999** [comment]
Dropping this into prebid.js appears to fix this:
var crypto = window.crypto || window.msCrypto;

**Comment by jsnellbaker** [comment]
@paulm999 Do you have a test page available where I can see this error occur?

**Comment by paulm999** [comment]
Yes, not a test page as such, as it's our live site but: https://www.netweather.tv/

**Comment by paulm999** [comment]
Actually, I think this is related to another module, not consent. I'll reach out to the authors of that.

**Comment by jsnellbaker** [comment]
Hi @paulm999 Thanks for update; please confirm when you can.  I'm continuing to double-check on our end just in case.

**Comment by GeneGenie** [comment]
> Hi @paulm999 Thanks for update; please confirm when you can. I'm continuing to double-check on our end just in case.

This repo does not contain any crypto usage
